photon-ml/src/integTest/scala/com/linkedin/photon/ml/GameModelGenerator.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linkedin.photon.ml
+
+import org.apache.log4j.Logger
+import org.apache.spark.SparkContext
+
+import com.linkedin.photon.ml.TaskType.TaskType
+import com.linkedin.photon.ml.model.{Coefficients, FixedEffectModel, GAMEModel, RandomEffectModel}
+import com.linkedin.photon.ml.supervised.classification.LogisticRegressionModel
+import com.linkedin.photon.ml.supervised.regression.LinearRegressionModel
+
+/**
+ * A generator of Game models.
+ *
+ * The models generated are random, within certain parameters that the user can specify (e.g., number of random effects,
+ * number of features in each random effect, ...).
+ */
+class GameModelGenerator {
+
+  private val log : Logger = Logger.getLogger(classOf[GameModelGenerator])
+  private val rng = scala.util.Random
+
+  private def randInt(M: Int) = 1 + math.abs(rng.nextInt) % M
+
+  private def randSparseVector(M: Int) = {
```

Comment:
See: https://github.com/linkedin/photon-ml/blob/master/photon-test/src/main/scala/com/linkedin/photon/ml/test/CommonTestUtils.scala

This would be better moved there / unified with the existing vector generators.